configure.ac

```diff
@@ -562,7 +562,7 @@ AC_ARG_ENABLE(weak-symbols,
 AC_ARG_ENABLE(qmpi,
 	AC_HELP_STRING([--qmpi],
```

Comment:
I think the help string need say `--enable-qmpi`

Reply:
Yep. Fixed.